waspc/cli/exe/Main.hs

```diff
@@ -102,7 +105,7 @@ printUsage =
         title "  IN PROJECT",
         cmd "    start                 Runs Wasp app in development mode, watching for file changes.",
         cmd "    db <db-cmd> [args]    Executes a database command. Run 'wasp db' for more info.",
-        cmd "    clean                 Deletes all generated code and other cached artifacts. Wasp equivalent of 'have you tried closing and opening it again?'.",
+        cmd "    clean                 Deletes all generated code and other cached artifacts, including dev db. Wasp equivalent of 'have you tried closing and opening it again?'.",
```

Comment:
Do we want clean to empty the DB as well? I feel that could be a bit surprising. I would vote for clean just deleting .wasp artifacts, and leaving DB clean to `wasp db reset`.

Reply:
Aha, I see later on about where the Docker DB data lives (.wasp) and how it was designed to behave similarly to SQLite. Ok, I think this makes sense. I still feel it may catch some people by surprise, but we can see!

Reply:
Ok, what I ended up doing was not deleting the db when it is postgres db, but sqlite one will end up deleted on `wasp clean`. 
It is just the easiest way to do it and I think in practice it will work just fine.
Only improvement could be to also void deleting sqlite on wasp clean, but that would be quite some work and is old behaviour anyway so I wouldn't worry about it for now.